src/components/search/Search.jsx

```diff
@@ -8,6 +9,5 @@ const Search = (props) => {
       <ResultSection ></ResultSection> */}
     </div>
   );
-
 };
-export default Search;
\ No newline at end of file
+export default Search;
```

Comment:
Added a newline in accordance with the [eol-last](https://eslint.org/docs/rules/eol-last) rule.